templates/trivia_qa/rc/templates.yaml

```diff
@@ -0,0 +1,99 @@
+dataset: trivia_qa
+subset: rc
+templates:
+  014c221a-ed54-4a1d-bc67-2ded6ea8c528: !Template
+    id: 014c221a-ed54-4a1d-bc67-2ded6ea8c528
+    jinja: "{% if answer.type == \"WikipediaEntity\" %} \nI've always wondered: {{question}}\
+      \ {{\"\\n\"}} Can you answer from wikipedia? You can try multiple times. |||\
+      \ {% for v in answer.aliases -%} Answer: {{v}} {% endfor -%}\n{% endif %}"
+    name: context_self_description_wikipedia_multiple_answer
+    reference: Ask question by self description from wikipedia
+    task_template: true
```

Comment:
you can remove this one, i am very not sure of the signal it brings and it doesn't make sense from the evaluation perspective

Reply:
same for `question_multiple_valid_answer`

Reply:
same for `question_with_instruction_multiple_valid_answer`

Reply:
and `wikipedia_contextual_question_multiple_answer`

Reply:
and `wikipedia_question_multiple_answer`